cmd/nknd/commands/root.go

```diff
@@ -43,6 +43,7 @@ import (
 	"github.com/nknorg/nnet/overlay/chord"
 	"github.com/rdegges/go-ipify"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
```

Comment:
`imported and not used: "github.com/spf13/pflag"` in my work tree.

Reply:
that's right. strange my vscode didnt notify me here. thanks.

Reply:
yes. I missed that one.